src/main/java/org/embulk/gradle/embulk_plugins/EmbulkPluginsPlugin.java

```diff
@@ -65,17 +65,8 @@ public void apply(final Project project) {
 
         final Configuration runtimeConfiguration = project.getConfigurations().getByName("runtime");
 
-        // It must be a non-detached configuration to be mapped into Maven scopes by Conf2ScopeMapping.
-        final Configuration alternativeRuntimeConfiguration = project.getConfigurations().maybeCreate("embulkPluginRuntime");
-
-        // It must be configured before evaluation (not in afterEvaluate).
-        replaceConf2ScopeMappings(project, runtimeConfiguration, alternativeRuntimeConfiguration);
```

Comment:
why is this piece is no longer needed?

Reply:
This configulation is for `uploadArchives`, which is unsuppoerted in this PR. The reason is written in the PR description.

Reply:
Instead of `uploadArchives` (`maven` plugin), `publishing` (`maven-publish` plugin) takes place. We are going to follow it.